Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                string name = PrintAndReadString("First Name: ");
+
+                string sureName = PrintAndReadString("Last Name: ");
+
+                int age = PrintAndReadInt("Age: "); ;
+
+                float weight = PrintAndReadFloat("Weight: ");
+
+                float height = PrintAndReadFloat("Height: ");
```

Comment:
Don;t write an empty line in between all. Put the related code close- unrelated, further away.